ui.tests/test-module/specs/ruleeditor/customFunction.runtime.spec.js

```diff
@@ -63,8 +63,9 @@ describe('Form with custom functions configured in client lib', () => {
         cy.get(`#${textbox1}`).find("input").should('have.value', "test")
     })
 
-    if (cy.af.isLatestAddon() && toggle_array.includes("FT_FORMS-11541")) {
-        it("should submit custom formData on button click", () => {
+
+    it("should submit custom formData on button click", () => {
```

Comment:
Can you write a test case for all the old forms, where the submitForm API has custom:submitSuccess and custom:submitError as arguments ?